Extensions/Video/JsExtension.js

```diff
@@ -0,0 +1,775 @@
+      .addIncludeFile("Extensions/Video/videoruntimeobject-pixi-renderer.js");
+
+    object
+      .addAction(
+        "Play",
+        _("Play an video"),
+        _("Play an video"),
+        _("Play the video of : _PARAM0_"),
+        "",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("play");
+
+    object
+      .addAction(
+        "Pause",
+        _("Pause an video"),
+        _("Pause an video"),
+        _("Pause the video on my object : _PARAM0_"),
+        "",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("pause");
+
+    object
+      .addAction(
+        "Loop",
+        _("Loop an video"),
+        _("Loop an video"),
+        _("Loop the video on my object : _PARAM0_ is on : _PARAM1_"),
+        "",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .addParameter("yesorno", _("Loop or not"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("setLoop");
+
+    object
+      .addAction(
+        "Mute",
+        _("Mute an video"),
+        _("Mute an video"),
+        _("Mute the video on my object : _PARAM0_ is on : _PARAM1_"),
+        "",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .addParameter("yesorno", _("Mute or not"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("mute");
+
+    object
+      .addAction(
+        "SetTime",
+        _("Set time (in seconds)"),
+        _("Set time (in seconds)"),
+        _("Set time of my object : _PARAM0_ is _PARAM1_ _PARAM2_ seconds"),
+        "",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .addParameter("operator", _("Modification's sign"), "", false)
+      .addParameter("expression", _("Time in seconds"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("setCurrentTime")
+      .setManipulatedType("number")
+      .setGetter("getCurrentTime");
+
+    object
+      .addAction(
+        "SetVolume",
+        _("Set volume (in %)"),
+        _("Set volume (in %)"),
+        _("Do _PARAM1_ _PARAM2_ % to the volume of _PARAM0_"),
+        "",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
```

Comment:
Let's avoid "Choose a" in the description of parameters. Parameters should just be description of what they are.

"Choose a video object" => "Video object"